src/main/java/core/mate/academy/model/ExcavatorProducer.java

```diff
@@ -0,0 +1,12 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer<Excavator> {
+    @Override
+    public List<? extends Machine> get() {
+        Machine machine = new Excavator();
```

Comment:
Consider creating a local variable for the list of machines. This can help make the code more readable and maintainable. For example: `List<Excavator> machines = List.of(new Excavator()); return machines;`